commit e64432ac6dde3bf6943d8fc929241d57a961a673
Author: Isaac Dunham <ibid.ag@gmail.com>
Date:   Tue Nov 18 10:26:22 2014 -0800

    shut up gcc: data_start is stored to, not used

diff --git a/archivers/zip.c b/archivers/zip.c
index 0e65453..611198d 100644
--- a/archivers/zip.c
+++ b/archivers/zip.c
@@ -1113,7 +1113,7 @@ static void *ZIP_openArchive(const char *name, int forWriting)
 {
     void *in = NULL;
     ZIPinfo *info = NULL;
-    PHYSFS_uint32 data_start;
+    PHYSFS_uint32 data_start = data_start;
     PHYSFS_uint32 cent_dir_ofs;
 
     BAIL_IF_MACRO(forWriting, ERR_ARC_IS_READ_ONLY, NULL);
